src/particle.h

```diff
@@ -78,6 +79,7 @@ class Particle : protected Pointers {
     int flag;               // used for migration status
     double dtremain;        // portion of move timestep remaining
     double weight;          // particle or cell weight, if weighting enabled
+    double g;               // particle weight (= FNUM if DSMC used)
```

Comment:
We typically discourage adding new members to the `OnePart` struct, prefer using the `custom` feature instead. However I'm open to @sjplimp's opinion on this.

Also using `g` for a variable is not descriptive and self obvious (at least to me), is there a different longer name that would be more descriptive?